inference-engine/tests/functional/plugin/cpu/single_layer_tests/normalize.cpp

```diff
@@ -90,10 +88,74 @@ const auto fusingMultiplyNoSharedChannel = fusingSpecificParams{std::make_shared
+                SizeVector secondMultInShape(1, 1);
+                auto secondMultInput = builder::makeConstant(ngPrc, Shape(secondMultInShape), std::vector<float>{}, true);
+                return std::make_shared<op::v1::Subtract>(inpNode, secondMultInput);
+            }, "Subtract(SharedChannel)"}}), {"Subtract"}};
+
+const auto fusingSubtractNoSharedChannel = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
+            {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+                SizeVector secondMultInShape(inpNode->get_shape().size(), 1);
+                secondMultInShape[1] = inpNode->get_shape()[1];
+                auto secondMultInput = builder::makeConstant(ngPrc, Shape(secondMultInShape), std::vector<float>{}, true);
+                return std::make_shared<op::v1::Subtract>(inpNode, secondMultInput);
+            }, "Subtract(NoSharedChannel)"}}), {"Subtract"}};
+
+const auto fusingDivideSharedChannel = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
+            {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+                SizeVector secondMultInShape(1, 1);
+                auto secondMultInput = builder::makeConstant(ngPrc, Shape(secondMultInShape), std::vector<float>{}, true);
+                return std::make_shared<op::v1::Divide>(inpNode, secondMultInput);
+            }, "Divide(SharedChannel)"}}), {"Divide"}};
+
+const auto fusingDivideNoSharedChannel = fusingSpecificParams{std::make_shared<postNodesMgr>(std::vector<postNodeBuilder>{
+            {[](std::shared_ptr<ngraph::Node> inpNode, const ngraph::element::Type& ngPrc, ngraph::ParameterVector& params){
+                SizeVector secondMultInShape(inpNode->get_shape().size(), 1);
+                secondMultInShape[1] = inpNode->get_shape()[1];
+                auto secondMultInput = builder::makeConstant(ngPrc, Shape(secondMultInShape), std::vector<float>{}, true);
+                return std::make_shared<op::v1::Divide>(inpNode, secondMultInput);
+            }, "Divide(NoSharedChannel)"}}), {"Divide"}};
+
 std::vector<fusingSpecificParams> fusingParamsSet {
         emptyFusingSpec,
         fusingMultiplySharedChannel,
-        fusingMultiplyNoSharedChannel
+        fusingMultiplyNoSharedChannel,
+        fusingAddSharedChannel,
+        fusingAddNoSharedChannel,
+        fusingSubtractSharedChannel,
+        fusingSubtractNoSharedChannel,
+        fusingDivideSharedChannel,
+        fusingDivideNoSharedChannel,
+        fusingPReluPerChannel,
+        fusingPReluPerTensor,
+        fusingRelu,
+        fusingElu,
+        fusingGelu,
+        fusingSigmoid,
+        fusingClamp,
+        fusingTanh,
+        fusingSwish,
+        fusingHSwish,
+        fusingMish,
+        fusingHSigmoid,
+        fusingAbs
```

Comment:
I am not sure we need to iterate over all activations since they are just elementwise operations. Maybe 2-3 is enough. I am just worrying about huge amount of test cases and increased validation time.

Reply:
only relu is validated